src/ObjectUtils/SymbolsFileTest.cpp

```diff
@@ -71,4 +71,46 @@ TEST(SymbolsFile, FailToCreateSymbolsFile) {
   EXPECT_THAT(invalid_file, HasError("File does not exist"));
 }
 
+TEST(GetAllowedSymbolFilenamesForModule, ElfFile) {
```

Comment:
`GetStandardSymbolFilenamesForModule`? Also below

Reply:
Thanks for finding that! done